src/main/scala/guessbook/GuessBook.scala

```diff
@@ -0,0 +1,111 @@
+package guessbook
+
+import com.github.tototoshi.csv.CSVReader
+import guesswho.GenderEnum
+
+import scala.util.{Failure, Random, Success, Try}
+
+object GuessBook {
```

Comment:
Do you think it would make sense to make this into a class? Then the getBooksFromCSV (anything to do with parsing files/reading data) can be moved to a different class and then injected in via DI. This would separate out these things which have different concerns